docs/GetStarted/getting_started_linux_vulkan.md

```diff
@@ -25,10 +25,8 @@ using [SwiftShader](https://swiftshader.googlesource.com/SwiftShader/) (a high
 performance CPU-based implementation of Vulkan).
 
 Vulkan API version > 1.2 is recommended where available. The
-`VK_KHR_timeline_semaphore` extension (part of Vulkan 1.2) is required, and the
-[Vulkan-ExtensionLayer](https://github.com/KhronosGroup/Vulkan-ExtensionLayer)
-project can be used to enable it for drivers (like SwiftShader) without native
-support.
+`VK_KHR_timeline_semaphore` extension (part of Vulkan 1.2) is required, though
```

Comment:
(There is no API > 1.2 at the moment. ;-P) 

I'd suggest to revise this sentence a bit like


IREE uses timeline semaphore in the Vulkan HAL implementation. Vulkan drivers implementing API version >= 1.2 is or the`VK_KHR_timeline_semaphore` extension is recommend. Otherwise, IREE will still be able to run by emulating timeline semaphores with other synchronization primitives, but it may come with performance cost.

Reply:
Thanks, revised a bit differently.

* I want to lead with "use this driver version because ..." rather than "because of ..., use this driver version". That should help when scanning the page.
* Keeping out some details (HAL, synchronization primitives) to keep this concise
* Included the mention about performance costs